src/calculator/service.ts

```diff
@@ -175,9 +175,23 @@ async function socialCredibility(userID: string) {
   }
 }
 
+function ScrappedSocialCredibility(followersCount: number, friendsCount: number){
```

Comment:
please, use camelCase, not PascalCase for functions/variables/methods.
PascalCase is ok on classes and interfaces.

Reply:
> We need tests for this. Check the comment that I left on #11
> Also, I see that the PR name is incorrect for this endpoint. Changing it.

I am pretty sure this is for social credibility, which is different from user credibility. Please confirm if not

Reply:
I was thinking about writing the tests after the mock data, but now I just recall that this does not need mock data :) Going to do the req changes

Reply:
> > We need tests for this. Check the comment that I left on #11
> > Also, I see that the PR name is incorrect for this endpoint. Changing it.
> 
> I am pretty sure this is for social credibility, which is different from user credibility. Please confirm if not
yes, you're right, but at the beginning the title was for tweet credibility. updating it to say that thi is for social